tests/queries/0_stateless/02421_simple_queries_for_opentelemetry.sh

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env bash
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CURDIR"/../shell_config.sh
+
+# This function takes 2 arguments:
+# $1 - query
+# $2 - trace id
+function executeQuery()
+{
+  ${CLICKHOUSE_CLIENT} --database=${CLICKHOUSE_DATABASE} --opentelemetry_start_trace_probability=1 --query_id $1 -nq "
+      ${2}
+  "
+}
+
+# For some quries, it's not able to know how many bytes/rows are read when tests are executed on CI,
+# so we only to check the db.statement only
+function check_query_span_query_only()
+{
+${CLICKHOUSE_CLIENT} -nq "
+    SYSTEM FLUSH LOGS;
+    SELECT attribute['db.statement']       as query
+    FROM system.opentelemetry_span_log
+    WHERE finish_date                      >= yesterday()
+    AND   operation_name                   = 'query'
+    AND   attribute['clickhouse.query_id'] = '${1}'
+    Format JSONEachRow
+    ;"
+}
+
+function check_query_span()
+{
+${CLICKHOUSE_CLIENT} -nq "
+    SYSTEM FLUSH LOGS;
+    SELECT attribute['db.statement']             as query,
+           attribute['clickhouse.read_rows']     as read_rows,
+           attribute['clickhouse.read_bytes']    as read_bytes,
+           attribute['clickhouse.written_rows']  as written_rows,
```

Comment:
The number of read and written bytes seems like an internal detail which is rather fragile and may break with future (unrelated!) commits. Suggest to not check it.

Reply:
OK, Let's keep rows only.